Lib/test/datetimetester.py

```diff
@@ -4361,6 +4361,13 @@ def test_fromtimestamp_lord_howe(self):
         self.assertEqual(t1.fold, 1)
 
 
+    def test_fromtimestamp_low_fold_detection(self):
```

Comment:
Should this test be marked as Windows-only?

Reply:
Not necessarily, while the bug is windows specific, if you look at the bpo thread you'll see there were no folds during this time. Therefore, this can just act as another unit test for fold detection on other platforms while being a regression test on windows.